types/index.d.ts

```diff
@@ -50,6 +50,7 @@ declare global {
     }
 
     namespace ChaiHttp {
+        interface Request extends request.SuperAgentRequest {}
```

Comment:
is there any particular reason you know of why this isn't `request.Request`?

`SuperAgentRequest` is an alias to it

Reply:
I might have been able to answer that question eighteen months ago. I'll take a look when I have some time to clone this repo and check it out.

Reply:
no worries. if you can catch up from main and double check that, we can probably merge this

if you don't have time, just let me know and ill do a new branch 👍